contracts/oracle/collateralization/CollateralizationOracle.sol

```diff
@@ -62,6 +61,9 @@ contract CollateralizationOracle is ICollateralizationOracle, CoreRef {
     ) CoreRef(_core) {
         _setOracles(_tokens, _oracles);
         _addDeposits(_deposits);
+
+        // Shared admin with other oracles
+        _setContractAdminRole(keccak256("ORACLE_ADMIN_ROLE"));
```

Comment:
Instead of creating more DAO actions, we could pre-compute the collateralization oracle guardian address and then pass that in the constructor and give the ORACLE_ADMIN_ROLE directly to that address.